packages/strapi-utils/lib/content-types.js

```diff
@@ -108,6 +112,60 @@ const isMediaAttribute = attr => {
   return (attr.collection || attr.model) === 'file' && attr.plugin === 'upload';
 };
 
+const getKind = obj => obj.kind || 'collectionType';
+
+const pickSchema = model => {
+  const schema = _.cloneDeep(
+    _.pick(model, ['connection', 'collectionName', 'info', 'options', 'attributes'])
+  );
+
+  schema.kind = getKind(model);
+  return schema;
+};
+
+const formatContentType = (
```

Comment:
can you rename to createContentType to avoid confusion with the format use to send in the api